homeassistant/components/sensor/tautulli.py

```diff
@@ -0,0 +1,133 @@
+"""
+A platform which allows you to get information from Tautulli.
+
+For more details about this component, please refer to the documentation at
+https://www.home-assistant.io/components/sensor.tautulli
+"""
+
+import logging
+import voluptuous as vol
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_API_KEY,
+                                 CONF_HOST,
+                                 CONF_MONITORED_VARIABLES,
+                                 CONF_PORT,
+                                 CONF_SSL,
+                                 STATE_UNAVAILABLE)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import (PLATFORM_SCHEMA)
```

Comment:
Please keep the import grouped. [`isort`](https://developers.home-assistant.io/docs/en/development_guidelines.html#ordering-of-imports) can help.